src/ai/backend/client/cli/admin/quota_scope.py

```diff
@@ -0,0 +1,195 @@
+import sys
+import uuid
+
+import click
+
+from ai.backend.cli.types import ExitCode
+from ai.backend.client.session import Session
+
+from ...output.fields import group_fields, quota_scope_fields, user_fields
+from ..pretty import print_error, print_fail
+from . import admin
+
+_user_query_fields = (
+    user_fields["uuid"],
+    user_fields["username"],
+)
+_project_query_fields = (
+    group_fields["id"],
+    group_fields["name"],
+)
+
+
+@admin.group()
+def quota_scope():
+    """Quota scope administration commands."""
+
+
+def _get_qsid_from_identifier(type: str, domain_name: str, identifier: str, session: Session):
+    match type:
+        case "user":
+            try:
+                user_id = uuid.UUID(identifier)
+            except ValueError:
+                # In case identifier is the user email
+                user_info = session.QuotaScope.get_user_info(
+                    domain_name=domain_name,
+                    email=identifier,
+                    fields=_user_query_fields,
+                )
+                if user_info is None:
+                    return None
+                user_qsid = f"user:{user_info['uuid']}"
+            else:
+                # In case identifier is the user ID
+                user_qsid = f"user:{user_id}"
+            return user_qsid
+
+        case "project":
+            try:
+                project_id = uuid.UUID(identifier)
+            except ValueError:
+                # In case identifier is the project name
+                project_info = session.QuotaScope.get_project_info(
+                    domain_name=domain_name,
+                    name=identifier,
+                    fields=_project_query_fields,
+                )
+                if project_info is None:
+                    return None
+                project_qsid = f"project:{project_info['id']}"
+            else:
+                # In case identifier is the project ID
+                project_qsid = f"project:{project_id}"
+            return project_qsid
```

Comment:
How about returning `ai.backend.common.types.QuotaScopeID` object rather than `str`?
Because the `QuotaScopeID` type defines serialization